src/components/Interviewee/Schedule/IntervieweeSchedule.tsx

```diff
@@ -56,19 +63,49 @@ const IntervieweeSchedule: React.FC<IntervieweeScheduleProps> = ({
           );
         })}
 
-        <Button
-          variant="outlined"
-          startIcon={<AddIcon />}
-          onClick={() =>
-            intervieweeAvaliabilityFields.setter({ type: CREATE_ACTION })
-          }
-        >
-          {copy.form.avaliabilityLabel}
-        </Button>
-        <br />
-        <Typography variant="body2" gutterBottom>
-          {copy.form.avaliabilityInfo}
-        </Typography>
+        <Box m={2} />
+        {/* TODO: Extract this Card as an Admonition component for reuse */}
+        <Card variant="elevation">
+          <CardContent>
+            <div
+              style={{
+                display: 'flex',
+                alignContent: 'center',
+              }}
+            >
+              <div
+                style={{
+                  display: 'flex',
+                  alignContent: 'center',
+                  alignItems: 'center',
+                }}
+              >
+                <InfoIcon />
+              </div>
+              <div style={{ width: '8px' }} />
+              <div
+                style={{
+                  display: 'flex',
+                  alignContent: 'center',
+                  alignItems: 'center',
+                }}
+              >
+                <Typography variant="h6">Note</Typography>
+              </div>
+            </div>
+            <div style={{ height: '8px' }} />
+            <Typography variant="body1">
+              In Proyecto Nutria we work with the Pacific Timezone, therefore
+              you should provide your availability in PT (usually CDMX time -2
+              hours)
+              <div style={{ height: '12px' }} />
+              <span style={{ fontWeight: 600 }}>Example:</span> If you set a
+              time between 2:00pm and 5:00pm then your interview will be between
```

Comment:
If I'm correct, sometimes the time zone difference is 1 hour and not 2 hours. Just FYI. :D

Reply:
Agree, I mentioned that it is "usually" 2 hours behind CDMX time, do you think we need a more strict note on that?

Reply:
I updated the text with a Link to a clock in PT time for reference, that should completely clarify any differences with the timezone